packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/EdgeStylePropertiesConfigurer.java

```diff
@@ -0,0 +1,499 @@
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+import org.eclipse.emf.common.util.Diagnostic;
+import org.eclipse.emf.common.util.EList;
+import org.eclipse.emf.common.util.Enumerator;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.util.EcoreUtil;
+import org.eclipse.emf.edit.domain.EditingDomain;
+import org.eclipse.sirius.components.collaborative.forms.services.api.IPropertiesDescriptionRegistry;
+import org.eclipse.sirius.components.collaborative.forms.services.api.IPropertiesDescriptionRegistryConfigurer;
+import org.eclipse.sirius.components.collaborative.validation.api.IValidationService;
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.emf.services.EditingContext;
+import org.eclipse.sirius.components.forms.SelectStyle;
+import org.eclipse.sirius.components.forms.components.SelectComponent;
+import org.eclipse.sirius.components.forms.description.AbstractControlDescription;
+import org.eclipse.sirius.components.forms.description.CheckboxDescription;
+import org.eclipse.sirius.components.forms.description.FormDescription;
+import org.eclipse.sirius.components.forms.description.GroupDescription;
+import org.eclipse.sirius.components.forms.description.PageDescription;
+import org.eclipse.sirius.components.forms.description.SelectDescription;
+import org.eclipse.sirius.components.representations.Failure;
+import org.eclipse.sirius.components.representations.GetOrCreateRandomIdProvider;
+import org.eclipse.sirius.components.representations.IStatus;
+import org.eclipse.sirius.components.representations.Success;
+import org.eclipse.sirius.components.representations.VariableManager;
+import org.eclipse.sirius.components.view.ArrowStyle;
+import org.eclipse.sirius.components.view.ColorPalette;
+import org.eclipse.sirius.components.view.EdgeStyle;
+import org.eclipse.sirius.components.view.FixedColor;
+import org.eclipse.sirius.components.view.LabelStyle;
+import org.eclipse.sirius.components.view.LineStyle;
+import org.eclipse.sirius.components.view.Style;
+import org.eclipse.sirius.components.view.UserColor;
+import org.eclipse.sirius.components.view.View;
+import org.eclipse.sirius.components.view.ViewPackage;
+import org.eclipse.sirius.components.view.emf.CustomImageMetadata;
+import org.eclipse.sirius.components.view.emf.ICustomImageMetadataSearchService;
+import org.eclipse.sirius.diagram.description.style.EdgeStyleDescription;
+import org.springframework.stereotype.Component;
+
+/**
+ * Customizes the properties view for some of the View DSL elements.
+ *
+ * @author mcharfadi
+ */
+@Component
+public class EdgeStylePropertiesConfigurer implements IPropertiesDescriptionRegistryConfigurer {
```

Comment:
There is duplication with helper methods from `NodeStylePropertiesConfigurer`. At least:
* createSimplePageDescription, createSimpleGroupDescription, getTargetObjectIdProvider
* createCheckbox
* getDiagnosticsProvider, kindProvider, messageProvider